web/controllers/user_controller.ex

```diff
@@ -24,7 +24,9 @@ defmodule Org.UserController do
   end
 
   def show(conn, %{"id" => id}) do
-    user = Repo.get!(User, id)
+    user = User
+            |> Repo.get!(id)
```

Comment:
I see that you've started to indent the pipe operator everywhere, but I have yet to see that in any other Elixir code, including the documentation. What is your preference? Personally I prefer no indent, and it is consistent with the community, but you could convince me otherwise if you feel strongly about it